pkg/config/install_requirements_test.go

```diff
@@ -356,3 +357,53 @@ func TestHelmfileRequirementValues(t *testing.T) {
 	assert.FileExists(t, file)
 	assert.FileExists(t, valuesFile, "file %s should exist", valuesFile)
 }
+
+func Test_LoadRequirementsConfig(t *testing.T) {
+	t.Parallel()
+
+	var gitPublicTests = []struct {
+		requirementsPath   string
+		createRequirements bool
+		loadSuccessful     bool
+	}{
+		{"a", false, false},
+		{"a/b", false, false},
+		{"a/b/c", false, false},
+		{"e", true, true},
+		{"e/f", true, true},
+		{"e/f/g", true, false},
+	}
+
+	for _, testCase := range gitPublicTests {
+		t.Run(testCase.requirementsPath, func(t *testing.T) {
+			dir, err := ioutil.TempDir("", "jx-test-load-requirements-config")
+			assert.NoError(t, err, "failed to create tmp directory")
```

Comment:
NIT: `require.NoError` can stop the unit test. In my experience set up should fail fast and assertion should be used after calling the function under test.

Reply:
ahhh, sorry missed this. that's the explanation first time around

Reply:
Good one. I agree with your reasoning. I prefer to fail fast as well. I'll remember this for the future.